swagger.json

```diff
@@ -208,347 +190,136 @@
-                      "example": "avatar.png",
-                      "format": "avatar"
-                    }
-                  }
+          "default": {
+            "description": "Unexpected error",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Error"
                 }
               }
             }
           }
         }
-      }
-    },
-    "/api/profile": {
+      },
       "patch": {
         "tags": ["Users"],
-        "summary": "Update profile of user",
         "description": "Update profile of user",
         "operationId": "updateProfile",
-        "produces": ["application/json"],
-        "parameters": [
-          {
-            "name": "body",
-            "in": "body",
-            "description": "user object",
-            "required": true,
-            "schema": {
-              "type": "object",
-              "properties": {
-                "email": {
-                  "type": "string",
-                  "description": "email of user",
-                  "example": "anymail@hotmail.com",
-                  "format": "email"
-                },
-                "username": {
-                  "type": "string",
-                  "description": "username of user",
-                  "example": "anyusername",
-                  "format": "username"
-                },
-                "firstName": {
-                  "type": "string",
-                  "description": "first name of user",
-                  "example": "anyfirstName",
-                  "format": "firstName"
-                },
-                "lastName": {
-                  "type": "string",
-                  "description": "last name of user",
-                  "example": "anylastName",
-                  "format": "lastName"
-                },
-                "location": {
-                  "type": "string",
-                  "description": "location of user",
-                  "example": "anylocation",
-                  "format": "location"
-                },
-                "phone": {
-                  "type": "string",
-                  "description": "phone of user",
-                  "example": "05446544533",
-                  "format": "phone"
-                },
-                "description": {
-                  "type": "string",
-                  "description": "description of user",
-                  "example": "anydescription",
-                  "format": "description"
-                },
-                "cv": {
-                  "type": "string",
-                  "description": "cv of user",
-                  "example": "cv.pdf",
-                  "format": "cv"
-                }
-              }
+        "requestBody": {
+          "description": "User to add",
+          "required": true,
+          "content": {
+            "application/json": {
+              "schema": "#/components/schemas/ProfileUser"
```

Comment:
thank you so much for the amazing efforts ! as for patch / profile we have agreed to go for one of them, Ngo or Volunteer in the body as well as the response